CoEpi/ui/onboarding/Constants.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+struct Constants {
+    
+    struct color {
```

Comment:
We probably should move this to a more central place, called `Theme` or similar. I imagine that these colors can be used in other places of the app.

Reply:
Good idea. I will give them better names also